CHANGES.md

```diff
@@ -1,3 +1,17 @@
+## Version 0.6.1
```

Comment:
A couple more notes:

It should be completely binary compatible (as verified by mima).

It's a little strange that the CHANGES.md PR is to the master branch. See the note in the original PR message about the current branch/release system being a little cumbersome. I'm hoping we can have a good system for this in the future.